packages/main/src/FileUploader.ts

```diff
@@ -317,6 +357,28 @@ class FileUploader extends UI5Element implements IFormInputElement {
 		}, "");
 	}
 
+	/**
+	 * Returns true if all files are below the maxFileSize, otherwise throws a "fileSizeExceeded" event if one file exceeds it.
+	 * @private
+	 */
+	_checkFileSize(files: FileList | null): boolean {
+		if (!(this.maxFileSize && files)) {
+			return true;
+		}
+
+		for (let i = 0; i < files.length; i++) {
+			const fileSize = (files[i].size / 1024) / 1024;
```

Comment:
extract 1024 in variable, so we don't have magic numbers. like megabyteSize or something like this. It will also help understanding why is devided twice.

Reply:
The File size is in bytes. The meaning of the literal 1024 couldn't be described exactly as a megabyte size.
This metric could be referred as kilo as in kilobyte or a chunk of some sort. IMO the magic number in this case carries semantic meaning by it self and it will be hard to think for a good variable name.

Reply:
The fileSize parameter in the event and the maxFileSize parameter are already documented as "file size in MB", and we know that [file.size is supposed to be in bytes](https://developer.mozilla.org/en-US/docs/Web/API/Blob/size). It would be logical that the line converts from bytes to MB

Reply:
We can have 2 variables in that case. "megabyteSize" was just an example.  If we have 2 will be even better, we will know that we have size in bytes, cause once we divide on kilobyte size, then on megabyte size.

Reply:
Actually magabyteSize should be correct as a variable name as this would be 1024 * 1024.

Reply:
The conversion now happens in `convertBytesToMegabytes` which should make it more descriptive.